DOCS-3373: Document 3DS2 for cardholder data payments (#2419)
MultiSafepay/docs: content/payment-methods/3ds2.md

- **Pricing:** Free
- **Activation:** Email <sales@multisafepay.com>

# Direct card payments

> When you collect cardholder data yourself, you also need to collect the contextual information from the customer's device (fingerprint). This is only relevant when creating `direct` card orders. **Not** when using [Payment Components](/docs/payment-components) or [Payment pages](/docs/payment-pages), where MultiSafepay collects the contextual information on your behalf.

Reviewer: ```suggestion
When you collect cardholder data yourself, you also need to collect other contextual information from the customer's device (fingerprint). This is only relevant for `direct` card orders, and **not** for [payment components](/docs/payment-components) or [payment pages](/docs/payment-pages), where MultiSafepay collects the fingerprint on your behalf.
```
